client/src/App.scss

```diff
@@ -254,6 +247,12 @@ Account row
 
 */
 
+.accountContainer {
```

Comment:
I notice that some of the past css classes use snake case instead of camel case. Is there a policy that the pattern app should follow?

Reply:
yes, i will standardize all of it.